ReachabilityUI/ReachabilityUI/Reachability/ReachabilityViewController.swift

```diff
@@ -26,11 +48,13 @@ class ReachabilityViewController: UIViewController {
     
     private var state: State! {
         didSet {
-            adjustLabelBasedOnConfiguration(state)
-            DispatchQueue.main.async {
-                if let animation = self.configuration.options[.animation] as? ReachabilityConfiguration.Animation {
-                    self.prepareForAnimatingPositionChange(self.state, animation: animation)
-                    self.window.bringSubviewToFront(self.view)
+            if state.shouldAnimate {
```

Comment:
shouldn't this be `newValue`? I mean...can't you end up in a loop here...potentially?

Reply:
Seems like I'm not allowed to use "newValue" here. Might only be in setters - right? 😅

Reply:
ah 🤦‍♂ yes, sorry